tests/test_file_handler.py

```
@@ -0,0 +1,57 @@
+    for t in threads:
+        t.start()
+    for t in threads:
+        t.join()
+    del handler
+    gc.collect()
+    data = path.read_text()
+    for i in range(10):
+        assert f"core [INFO] msg{i}" in data
+
+
+def test_file_handler_open_failure(tmp_path: Path) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)